### PR TITLE
Add Gerrit server info to contributing doc

### PR DESCRIPTION
**Pull request by anoo1**
Add steps to setup and push changes to the openbmc gerrit server.

Signed-off-by: Adriana Kobylak <anoo@us.ibm.com>

---
This change is [Reviewable](https://reviewable.io/reviews/openbmc/docs/30)

### CODE REVIEW

**Comment by amboar** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[contributing.md, line 109 \[r1\]](https://reviewable.io:443/reviews/openbmc/docs/30#-KKuaTugM4Jd3phqLBWX:-KKuaTugM4Jd3phqLBWY:57050519) ([raw file](https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md#L109)):*
> ```Markdown
> <openbmc_repo> is the current git repository you're working on, such as
> phosphor-rest-server:
> `remote add gerrit ssh://your_github_id@openbmc.gerrit/openbmc/openbmc_repo`
> ```

Bit of a nit-pick, but this should be `git remote add ...`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openbmc/docs/30)*

**Comment by amboar** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[contributing.md, line 107 \[r1\]](https://reviewable.io:443/reviews/openbmc/docs/30#-KKubE-wJxOxDNlno5Uw:-KKubE-wJxOxDNlno5Ux:1829612135) ([raw file](https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md#L107)):*
> ```Markdown
> 
> From your openbmc git repository, add a remote to the gerrit server, where
> <openbmc_repo> is the current git repository you're working on, such as
> ```

Don't forget that the markdown is rendered into HTML when viewed on github. If you look at[1] you'll see that the text '<openbmc_repo>' is missing due to the angle brackets.

[1] https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md

---

*[contributing.md, line 116 \[r1\]](https://reviewable.io:443/reviews/openbmc/docs/30#-KKubW-TTsgN5R266BHc:-KKubW-U3aF7LUDUHO4w:-198838341) ([raw file](https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md#L116)):*
> ```Markdown
> `scp -p -P 29418 your_github_id@gerrit.openbmc-project.xyz:hooks/commit-msg ${gitdir}/hooks`
> 
> To submit a change set, push to the gerrit server, where <gerrit> is the name
> ```

Angle brackets again here.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openbmc/docs/30)*

**Comment by amboar** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[contributing.md, line 107 \[r1\]](https://reviewable.io:443/reviews/openbmc/docs/30#-KKubE-wJxOxDNlno5Uw:-KKubmyUss1VHrgC-Low:924302030) ([raw file](https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md#L107)):*
<details><summary><i>Previously, amboar (Andrew Jeffery) wrote…</i></summary>
> Don't forget that the markdown is rendered into HTML when viewed on github. If you look at[1] you'll see that the text '<openbmc_repo>' is missing due to the angle brackets.
> 
> [1] https://github.com/openbmc/docs/blob/b0d149b2ce70c4b3db5a92dae9a5a7079532e912/contributing.md
</details>
Hah, turns out reviewable doesn't handle &lt; or &gt; properly either...

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openbmc/docs/30)*

**Comment by mdmillerii** [comment]
I would suggest using single underscore on each end to render it in _italics_, and still be `_readable_` in plane text.

**Comment by williamspatrick** [comment]
Per @kc8apf ...

We should add documentation on first time login.  User needs to log into webUI first to get an account on Gerrit server.  Also should go to https://gerrit.openbmc-project.xyz/plugins/github-plugin/static/account.html to ensure SSH keys are imported from Gerrit.  Without these two steps, command line doesn't work.

File: contributing.md [hunk 1]
```diff
@@ -84,6 +84,52 @@ The guidelines in the Linux kernel are very useful:
...
+phosphor-rest-server, and 'openbmc.gerrit' is the name of the Host previously added:
+
+  `git remote add gerrit ssh://your_github_id@openbmc.gerrit/openbmc/openbmc_repo`
+
+Obtain the git hook commit-msg to automatically add a Change-Id to the commit
+message, which is needed by gerrit:
+
+  `gitdir=$(git rev-parse --git-dir)`
+
+  `scp -p -P 29418 your_github_id@gerrit.openbmc-project.xyz:hooks/commit-msg ${gitdir}/hooks`
```
Comment (williamspatrick) [inline contributing.md]: Can change ``` your_github_id@gerrit.openbmc-project.xyz ``` to ``` openbmc.gerrit ```.

File: contributing.md [hunk 2]
```diff
@@ -84,6 +84,52 @@ The guidelines in the Linux kernel are very useful:
...
+        Hostname gerrit.openbmc-project.xyz
+        Port 29418
+        User your_github_id
+```
+
+From your openbmc git repository, add a remote to the gerrit server, where
+'openbmc_repo' is the current git repository you're working on, such as
+phosphor-rest-server, and 'openbmc.gerrit' is the name of the Host previously added:
+
+  `git remote add gerrit ssh://your_github_id@openbmc.gerrit/openbmc/openbmc_repo`
```
Comment (williamspatrick) [inline contributing.md]: Don't need ```your_github_id``` if you have the openbmc.gerrit alias (with User) in it.

**Pull request merged**